ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -270,21 +270,21 @@
+| {{ ethernet_interface.name }} | {{ description }} | *{{ type }} | {{ channel_group }} | *{{ ipv6_address }} | *{{ vrf }} | *{{ mtu }} | *{{ shutdown | lower }} | *{{ nd_ra_disabled | lower }} | {{ managed_config_flag | lower }} | *{{ ipv6_acl_in }} | *{{ ipv6_acl_out }} |
 {%                 endif %}
 {%             else %}
-{%                 if ethernet_interfaces[ethernet_interface].ipv6_address is arista.avd.defined or ethernet_interfaces[ethernet_interface].ipv6_enable is arista.avd.defined(true) %}
-{%                     set description = ethernet_interfaces[ethernet_interface].description | arista.avd.default("-") %}
-{%                     set type = ethernet_interfaces[ethernet_interface].type | arista.avd.default("switchport") %}
-{%                     set ipv6_address = ethernet_interfaces[ethernet_interface].ipv6_address | arista.avd.default("-") %}
-{%                     set vrf = ethernet_interfaces[ethernet_interface].vrf | arista.avd.default("default") %}
-{%                     set mtu = ethernet_interfaces[ethernet_interface].mtu | arista.avd.default("-") %}
-{%                     set shutdown = ethernet_interfaces[ethernet_interface].shutdown | arista.avd.default("-") %}
-{%                     set nd_ra_disabled = ethernet_interfaces[ethernet_interface].ipv6_nd_ra_disabled | arista.avd.default("-") %}
-{%                     set managed_config_flag = ethernet_interfaces[ethernet_interface].ipv6_nd_managed_config_flag | arista.avd.default("*-") %}
-{%                     set ipv6_acl_in = ethernet_interfaces[ethernet_interface].ipv6_access_group_in | arista.avd.default("-") %}
-{%                     set ipv6_acl_out = ethernet_interfaces[ethernet_interface].ipv6_access_group_out | arista.avd.default("-") %}
-| {{ ethernet_interface }} | {{ description }} | {{ type }} | - | {{ ipv6_address }} | {{ vrf }} | {{ mtu }} | {{ shutdown | lower }} | {{ nd_ra_disabled | lower }} | {{ managed_config_flag | lower }} | {{ ipv6_acl_in }} | {{ ipv6_acl_out }} |
+{%                 if ethernet_interface.ipv6_address is arista.avd.defined or ethernet_interface.ipv6_enable is arista.avd.defined(true) %}
+{%                     set description = ethernet_interface.description | arista.avd.default("-") %}
+{%                     set type = ethernet_interface.type | arista.avd.default("switchport") %}
+{%                     set ipv6_address = ethernet_interface.ipv6_address | arista.avd.default("-") %}
+{%                     set vrf = ethernet_interface.vrf | arista.avd.default("default") %}
+{%                     set mtu = ethernet_interface.mtu | arista.avd.default("-") %}
+{%                     set shutdown = ethernet_interface.shutdown | arista.avd.default("-") %}
+{%                     set nd_ra_disabled = ethernet_interface.ipv6_nd_ra_disabled | arista.avd.default("-") %}
+{%                     set managed_config_flag = ethernet_interface.ipv6_nd_managed_config_flag | arista.avd.default("*-") %}
```

Comment:
I think the * in the default value is added by mistake. I will remove it in a different PR.

Reply:
Let's fix these small typos now, or else we will forget!

Reply:
resolved in e78e9d7

Reply:
Fixed in PR #1661